Add steps for submitting Sourcegraph metric dumps to support (#6422)
sourcegraph/sourcegraph: doc/admin/troubleshooting.md

```

See [sourcegraph/sourcegraph#398](https://github.com/sourcegraph/sourcegraph/issues/398) for more information.

### Submitting a metrics dump

> When you encounter performance or instability problem with Sourcegraph, we may ask you to submit a metrics dump to us. This allows us to inspect the performance and health of various parts of your Sourcegraph instance in the past and can often be the most effective way for us to identify the cause of your issue.

Reviewer: When you encounter a performance or instability problem

> When you encounter performance or instability problem with Sourcegraph, we may ask you to submit a metrics dump to us. This allows us to inspect the performance and health of various parts of your Sourcegraph instance in the past and can often be the most effective way for us to identify the cause of your issue.

Author: ```suggestion
If you encounter performance or instability issues with Sourcegraph, we may ask you to submit a metrics dump to us. This allows us to inspect the performance and health of various parts of your Sourcegraph instance in the past and can often be the most effective way for us to identify the cause of your issue.
```

Reviewer: Yeah, that is better. We aim for perfection but if we fall short :-)
